app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntrySheet.kt

```diff
@@ -24,7 +25,7 @@ class BloodSugarEntrySheet : BottomSheetActivity(), BloodSugarEntryUi {
   }
 
   override fun hideBloodSugarErrorMessage() {
-    TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
+    bsErrorTextView.visibleOrGone(false)
```

Comment:
Can we not use abbreviations unless they have an accepted meaning in the domain, please? BP is a known abbreviation for Blood Pressure, so it is acceptable to use it sometimes instead of `BloodPressure`, but `BS` is not a standard abbreviation for Blood Sugar.